Assignment8/Assignment8/MainWindow.xaml.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+using System.Windows.Threading;
+
+namespace Assignment8
+{
+    /// <summary>
+    /// Interaction logic for MainWindow.xaml
+    /// </summary>
+    public partial class MainWindow : Window
+    {
+        private DispatcherTimer dispatcherTimer = new ();
+        private DateTime _StartTime = DateTime.Now;
+        private string _Description;
+        private bool _IsStopped = true;
+        private int lap = 1;
+        public MainWindow()
+        {
+            InitializeComponent();
+
+            dispatcherTimer.Tick += new EventHandler(dispatcherTimer_Tick);
```

Comment:
Event handler can be simplified.
```suggestion
            dispatcherTimer.Tick += dispatcherTimer_Tick;
```